Project/cypress/e2e/claim.spec.cy.ts

```diff
@@ -0,0 +1,85 @@
+
+import { id, submitClaim } from "../support/helper/claim";
+import { addEvent, deleteEvent } from "../support/helper/event";
+import { addExpense, deleteExpense } from "../support/helper/expense";
+import { AssignClaim } from "../support/pageobjects/AssignClaim";
+import { Employee } from "../support/pageobjects/Employee";
+
+
+let empNumber: string;
+let eventName = "EventART";
+let eventId: string;
+let expenseName = "ExpenseART";
+let expenseId: string;
+let amount = "5";
+describe("", () => {
+    beforeEach(function () {
+        cy.visit("/web/index.php/auth/login");
+        cy.fixture('employee').as('EmpInfo')
+        cy.login("Admin", "admin123").then(() => {
+            cy.get('@EmpInfo').then((infoData: any) => {
+                Employee.addEmployee(infoData.employees[0].firstName, infoData.employees[0].lastName)
+                    .then(async (response) => {
+                        empNumber = await response.body.data.empNumber
```

Comment:
instead, you can return the empNumber from the addEmployee function itself, and don't store the value in a const and use it in the same scope, so that you don't have to add async and await